exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,260 @@
+{  
+    "exercise":"yacht",
+    "version":"1.0.0",
+    "comments":[  
+        "The dice are represented always as a list of exactly five integers",
+        "with values beteen 1 and 6 inclusive. The category is an string.",
+        "the categories are 'ones' to 'sixes' then",
```

Comment:
the word "then" is repeated on the end of this line and the beginning of the next. One of the two should be removed.